public_papers/PRFE_results/Private Reach & Frequency Estimators Evaluation Results.md

```diff
@@ -0,0 +1,2273 @@
+### _[Draft For WFA | Review In Progress]_
+
+# Private Reach & Frequency Estimators Evaluation Results
+
+<br>
+<table align="center">
+  <tr>
+   <td align="center"><strong>Craig Wright</strong>
+   </td>
+   <td align="center"><strong>Aaron An</strong>
+   </td>
+   <td align="center"><strong>Xichen Huang</strong>
+   </td>
+  </tr>
+  <tr>
+  </tr>
+  <tr>
+   <td align="center">Google, LLC
+   </td>
+   <td align="center">Facebook Inc.
+   </td>
+   <td align="center">Google, LLC
+   </td>
+  </tr>
+</table>
+
+<br>
+
+<center>
+  Additional contributors to evaluation code base and design in Appendix G
```

Comment:
Do you want to add hyperlinks to Appendices here and where they appear in the rest of the text?

Reply:
Good call Craig! I will add this before merge.